nms/app/packages/magmalte/app/views/tracing/TraceStartDialog.js

```diff
@@ -61,6 +61,9 @@ const useStyles = makeStyles(_ => ({
       background: colors.primary.mirage,
     },
   },
+  addSubscriberDialog: {
```

Comment:
I think we should set a min-height: '480px' to the dialog content in default.js styles

Reply:
Oh good call. Thanks!